geode-dunit/src/main/java/org/apache/geode/test/dunit/internal/ProcessManager.java

```diff
@@ -183,36 +183,55 @@ public void run() {
     ioTransport.start();
   }
 
+  private String removeModulesFromPath(String classpath, String... modules) {
+    for (String module : modules) {
+      classpath = removeModuleFromGradlePath(classpath, module);
+      classpath = removeModuleFromEclipsePath(classpath, module);
+    }
+    return classpath;
+  }
+
+  private String removeModuleFromEclipsePath(String classpath, String module) {
+    String buildDir = File.separator + module + File.separator + "out" + File.separator;
+
+    String mainClasses = buildDir + "production";
+
+    classpath = removeFromPath(classpath, mainClasses);
+    return classpath;
+  }
+
+  private String removeModuleFromGradlePath(String classpath, String module) {
+    String buildDir = File.separator + module + File.separator + "build" + File.separator;
+
+    String mainClasses = buildDir + "classes" + File.separator + "java" + File.separator + "main";
+    classpath = removeFromPath(classpath, mainClasses);
+
+    String libDir = buildDir + "libs";
+    classpath = removeFromPath(classpath, libDir);
+
+    String mainResources = buildDir + "resources" + File.separator + "main";
+    classpath = removeFromPath(classpath, mainResources);
+
+    String generatedResources = buildDir + "generated-resources" + File.separator + "main";
+    classpath = removeFromPath(classpath, generatedResources);
+
+    return classpath;
+  }
+
   private String[] buildJavaCommand(int vmNum, int namingPort, String version) {
-    String cmd = System.getProperty("java.home") + File.separator + "bin" + File.separator + "java";
+    String cmd = System.getProperty("java.home") + File.separator
+        + "bin" + File.separator + "java";
```

Comment:
Just a reminder that, by popular request, `spotless` will no longer remove extra whitespace within a line, even if the result will no longer exceed the 80-width limit.  For instance, this line's diff isn't necessary for spotless (but I assume it was at some point during your iteration).